terraform/environments/analytical-platform-compute/quicksight/data.tf

```diff
@@ -0,0 +1,33 @@
+data "aws_availability_zones" "available" {}
+
+data "aws_iam_session_context" "current" {
+  arn = data.aws_caller_identity.current.arn
+}
+
+data "aws_ssoadmin_instances" "main" {
+  provider = aws.sso-readonly
+}
+
+data "aws_security_group" "quicksight_shared_vpc_security_group" {
+  filter {
+    name   = "tag:Name"
+    values = ["quicksight-shared-vpc"]
+  }
+}
```

Comment:
Is this required? `module.quicksight_shared_vpc_security_group` exists in this component so you reference it directly